Include/RmlUi/Core/Types.h

```diff
@@ -29,22 +29,15 @@
 #ifndef RMLUI_CORE_TYPES_H
 #define RMLUI_CORE_TYPES_H
 
-#include <string>
 #include <cstdlib>
 #include <memory>
-#include <utility>
-#include <vector>
 
 #include "Traits.h"
 
-#ifdef RMLUI_NO_THIRDPARTY_CONTAINERS
-#include <set>
-#include <unordered_set>
-#include <unordered_map>
+#ifdef RML_USER_CONFIG_FILE
+#include RML_USER_CONFIG_FILE
```

Comment:
```suggestion
#ifdef RMLUI_USER_CONFIG_FILE
#include RMLUI_USER_CONFIG_FILE
```
(also in CMake if needed)